bedrock/firefox/templates/firefox/personal.html

```diff
@@ -0,0 +1,55 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+ {% extends "base-resp.html" %}
+
+{% block page_title_prefix %}{{_('Firefox Web browser')}} - {% endblock %}
```

Comment:
Is lowercase `browser` wanted? We had this string in the past, but it was ` Firefox Web Browser `